app/uk/gov/hmrc/thirdpartyapplication/controllers/Model.scala

```diff
@@ -51,7 +51,7 @@ case class OverridesResponse(overrides: Set[OverrideFlag])
 
 case class UpdateRateLimitTierRequest(rateLimitTier: String)
 
-case class DeleteApplicationRequest(gatekeeperUserId: String, requestedByEmailAddress: String)
+case class DeleteApplicationRequest(gatekeeperUserId: Option[String], requestedByEmailAddress: Option[String])
```

Comment:
Is this an a or b situation ?

Reply:
That change was reverted in another commit. Those don't need to be optional.